### PR TITLE
[Revival] Emaggable Soundsynth. Corn Syrup has never tasted so sweet.

### PR DESCRIPTION
**Pull request by Eneocho**
### What this does
Sound Synthesizers can be emagged, which gives them access to some less than theatrical sound effects. Code from #32787.
Does not add the bwoink or discord sound. We should not let the OOC issues with the previous attempts at this PR deny us a genuinely good feature.
### Why it's good
Clowns and mimes are theatrical beings, masters of acting. Let them perform the greatest farce and gaslight the entire station into believing a blob spawned in maintenance.

:cl:
 * rscadd: Sound synths can be emagged, which changes their icon and gives them access to special sound effects.

### CODE REVIEW

File: code/game/objects/items/devices/sound_synth.dm [hunk 1]
```diff
@@ -68,6 +83,18 @@
...
+/obj/item/device/soundsynth/emag_act(mob/user)
+	spark(src,5,FALSE)
+	if(!emagged)
+		emagged = 1
+		sound_list = sound_list_emagged + sound_list
+		selected_sound = "sound/effects/superfart.ogg"
+
+/obj/item/device/soundsynth/examine(mob/user)
+	..()
+	if(emagged)
```
Comment (boy2mantwicethefam) [inline code/game/objects/items/devices/sound_synth.dm]: Make this a "holding item" check so that only those who possess it can see it

Comment (Eneocho) [reply code/game/objects/items/devices/sound_synth.dm]: Can you elaborate please? I just copypasted pedr's code.

Comment (Eneocho) [reply code/game/objects/items/devices/sound_synth.dm]: Would `If(user.is_holding_item(src) && emagged)` work?

Comment (boy2mantwicethefam) [reply code/game/objects/items/devices/sound_synth.dm]: Yes, but put emagged check first for micro-optimization

Comment (Eneocho) [reply code/game/objects/items/devices/sound_synth.dm]: Done.

**Comment by PrimeDSS13** [comment]
I still think it's a bit shittery but eh whatever sure why not

**Comment by Kurfursten** [comment]
I could give you an emagged sprite if you won't codersprite yourself one. It would be nice if it was noticeably different.

**Comment by Eneocho** [comment]
> I could give you an emagged sprite if you won't codersprite yourself one. It would be nice if it was noticeably different.

That would be appreciated.

**Comment by Kurfursten** [comment]
sent

**Pull request merged**